sdk/storage/azure-storage-files-datalake/test/ut/datalake_file_client_test.cpp

```diff
@@ -217,8 +217,8 @@ namespace Azure { namespace Storage { namespace Test {
 
   TEST_F(DataLakeFileClientTest, FileDataActions)
   {
-    const int32_t bufferSize = 4 * 1024; // 4KB data size
-    std::vector<uint8_t> buffer(bufferSize, 'x');
+    const int32_t bufferSize = 10;
```

Comment:
Why change the size from 4 KB down to 10?

Reply:
to reduce recording size.

The test case has nothing to do with the blob size, so it's safe to do so.